<launch>
    <node pkg="nodelet" type="nodelet" name="standalone_nodelet" args="manager"
          output="screen"/>
    <node pkg="nodelet" type="nodelet" name="galaxy_camera"
          args="load galaxy_camera/GalaxyCameraNodelet standalone_nodelet"
          output="screen">
        <param name="image_width" value="1296"/><!--2592 -->
        <param name="image_height" value="972"/><!--1944-->
        <param name="image_offset_x" value="648"/>
        <param name="image_offset_y" value="486"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="pitch_camera"/>
    </node>

    <node ns="galaxy_camera" name="image_proc" pkg="image_proc"
          type="image_proc" respawn="true"/>
</launch>
